packages/google_maps_flutter/google_maps_flutter/example/ios/Podfile

```diff
@@ -29,6 +29,9 @@ flutter_ios_podfile_setup
 
 target 'Runner' do
   flutter_install_all_ios_pods File.dirname(File.realpath(__FILE__))
+  target 'RunnerTests' do
+    inherit! :search_paths
+  end
```

Comment:
`RunnerTests` needs to link on Flutter and plugins.  `RunnerUITests` do not since they interact with the UI blackbox.